paddle/fluid/operators/reader/lod_tensor_blocking_queue.h

```diff
@@ -65,21 +63,6 @@ class LoDTensorBlockingQueue {
   inline bool IsClosed() const { return queue_.IsClosed(); }
 
  private:
-  void CheckDims(
```

Comment:
Do not check dims every batch, since there are some model will change input dimension every batch.